libcheckout

```diff
@@ -1,4 +1,14 @@
 function checkout() {
+  if [ -z $SEMAPHORE_GIT_BRANCH ] || [ -z $SEMAPHORE_GIT_URL ] || [ -z $SEMAPHORE_GIT_DIR ] || [ -z $SEMAPHORE_GIT_SHA ]; then
+    echo "[CHECKOUT ERROR] The following variables need to be exported:"
+    echo "SEMAPHORE_GIT_URL"
+    echo "SEMAPHORE_GIT_DIR"
+    echo "SEMAPHORE_GIT_BRANCH"
+    echo "SEMAPHORE_GIT_SHA"
+
+    exit 1
+  fi
```

Comment:
You're right. I tweaked this. Can you have a look? 🙏

Reply:
Much better.

In the ideal case, the error would be `[error] SEMAPHORE_GIT_URL and SEMAPHORE_GIT_SHA can't be blank.`

But this will also do the work. 👍